Source/DetailsInputView.swift

```diff
@@ -45,8 +45,8 @@ import UIKit
         value.font = CheckoutTheme.font
 
         #if TARGET_INTERFACE_BUILDER
-        self.button = UIButton(type: UIButtonType.contactAdd)
-        value!.text = "value"
+        self.button = UIButton(type: .contactAdd)
```

Comment:
Can we add any test to check this ? change

Reply:
It's a compilation issue so no functionality has changed, we'd be unable to test this.